mapie/tests/test_utils.py

```diff
@@ -71,8 +79,8 @@ def test_check_null_weight_with_zeros() -> None:
 @pytest.mark.parametrize("estimator", [LinearRegression(), DumbEstimator()])
 @pytest.mark.parametrize("sample_weight", [None, np.ones_like(y_toy)])
 def test_fit_estimator(
-    estimator: Any, sample_weight: Optional[ArrayLike]
-) -> None:
+        estimator: Any,
+        sample_weight: Optional[ArrayLike]) -> None:
```

Comment:
I believe this is the correct indentation?

def test_fit_estimator(
        estimator: Any,
        sample_weight: Optional[ArrayLike]
) -> None: